### DISCUSSION TITLE
recipes/breads/sourdough-crumpets/

### DISCUSSION THREAD

**Discussion by giscus** [nicholaswilde/recipes#1081] [Comments]
# recipes/breads/sourdough-crumpets/

A collection of my recipes.

https://nicholaswilde.io/recipes/breads/sourdough-crumpets/

**Comment by nicholaswilde** [comment]
- Grease tins just before cooking.
- Cover with metal bowl when cooking on first side.
- Start cooking soon after adding baking soda.
